File: MainApp/src/main/java/com/sleticalboy/dailywork/weight/view/PageIndicator.java
package com.sleticalboy.dailywork.weight.view;

import androidx.viewpager.widget.ViewPager;

/**
 * Created on 18-3-15.
 *
 * @author leebin
 * @description 页面指示器
 */
public interface PageIndicator {

    /**
     * @param viewPager
     * @param initialPos
     */
    void setupWithViewPager(ViewPager viewPager, int initialPos);

    /**
     * @param pageIndex
     */
    void setCurrentPage(int pageIndex);

    /**
     *
     */
    void notifyDataSetChanged();
}

File: MainApp/src/main/java/com/sleticalboy/weight/view/PageIndicator.kt
package com.binlee.weight.view

import androidx.viewpager.widget.ViewPager

/**
 * Created on 18-3-15.
 *
 * @author leebin
 * @description 页面指示器
 */
interface PageIndicator {
  /**
   * @param viewPager
   * @param initialPos
   */
  fun setupWithViewPager(viewPager: ViewPager?, initialPos: Int)

  /**
   * @param pageIndex
   */
  fun setCurrentPage(pageIndex: Int)

  /**
   *
   */
  fun notifyDataSetChanged()
}